Library/Homebrew/cmd/tap.rb

```diff
@@ -76,6 +76,6 @@ def tap
 
   def force_auto_update?(args:)
     # if no relevant flag is present, return nil, meaning "no change"
-    true if args.force_auto_update?
+    args.force_auto_update?
   end
```

Comment:
I'd just remove this function altogether and invoke `args.force_auto_update?` directly.